libgadget/fof.c

```diff
@@ -150,26 +149,20 @@ void fof_fof(ForceTree * tree)
         HaloLabel[i].Pindex = i;
     }
     /* Fill FOFP_List of primary */
-    t0 = second();
     fof_label_primary(tree);
-    t1 = second();
 
-    message(0, "group finding took = %g sec\n", timediff(t0, t1));
+    message(0, "Group finding done.\n");
```

Comment:
I think the main reason it were there was because if the code crashes inside a step we never see the wall-clock report of that step. Perhaps we shall add a signal handler that saves the current per rank wall-clock report at crash?

Reply:
We do still know how long the group finding took, because we see in the output log a timestamp for each message and we can do sums. If we see:

[10.01] Starting FoF
[10000.5] Group finding done

then we know what that means.